<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12706" started-at="2019-10-15T19:39:27 IST" finished-at="2019-10-15T19:39:40 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12706" started-at="2019-10-15T19:39:27 IST" finished-at="2019-10-15T19:39:40 IST">
      <class name="ApiTests.Fb_Ads_From_Fb_Url">
        <test-method status="PASS" signature="setbaseUri()[pri:0, instance:ApiTests.Fb_Ads_From_Fb_Url@57855c9a]" name="setbaseUri" is-config="true" duration-ms="328" started-at="2019-10-15T19:39:27 IST" finished-at="2019-10-15T19:39:28 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setbaseUri -->
        <test-method status="FAIL" signature="Verify_Fb_Ads_From_Fb_Url()[pri:0, instance:ApiTests.Fb_Ads_From_Fb_Url@57855c9a]" name="Verify_Fb_Ads_From_Fb_Url" duration-ms="12346" started-at="2019-10-15T19:39:28 IST" finished-at="2019-10-15T19:39:40 IST">
          <exception class="org.json.simple.parser.ParseException">
            <full-stacktrace>
              <![CDATA[Unexpected character (i) at position 0.
at org.json.simple.parser.Yylex.yylex(Yylex.java:610)
at org.json.simple.parser.JSONParser.nextToken(JSONParser.java:269)
at org.json.simple.parser.JSONParser.parse(JSONParser.java:118)
at org.json.simple.parser.JSONParser.parse(JSONParser.java:81)
at org.json.simple.parser.JSONParser.parse(JSONParser.java:75)
at ApiTests.Fb_Ads_From_Fb_Url.Verify_Fb_Ads_From_Fb_Url(Fb_Ads_From_Fb_Url.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.simple.parser.ParseException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Fb_Ads_From_Fb_Url -->
      </class> <!-- ApiTests.Fb_Ads_From_Fb_Url -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
